daffodil-cli/src/it/scala/org/apache/daffodil/unparsing/TestCLIUnparsing.scala

```diff
@@ -244,19 +255,20 @@ class TestCLIunparsing {
     val schemaFile = Util.daffodilPath("daffodil-test/src/test/resources/org/apache/daffodil/section00/general/generalSchema.dfdl.xsd")
     val (testSchemaFile) = if (Util.isWindows) (Util.cmdConvert(schemaFile)) else (schemaFile)
 
-    val shell = Util.start("", true)
+    val shell = Util.start("")
 
     try {
 
-      shell.sendLine(String.format("""echo '<tns:e1 xmlns:tns="http://example.com">Hello</tns:e1>' | %s -v unparse -s %s --root e1""", Util.binPath, testSchemaFile))
-      shell.expect(contains("[info]"))
+      shell.sendLine(String.format("""echo "<tns:e1 xmlns:tns='http://example.com'>Hello</tns:e1>" | %s -v unparse -s %s --root e1""", Util.binPath, testSchemaFile))
```

Comment:
Hmm, looks like using the double quotes still doesn't work. I guess echo on windows will still echo double quotes? I think maybe using our `Util.echoN` function will fix this, since that echos out data a little differently and I think will strip off the double quotes? So maybe something like this:

```scala
shell.sendLine(String.format(Util.echoN("\"<tns:e1 xmlns:tns='http://example.com'>Hello</tns:e1>\"") + "| %s -v unparse ....", ...))
```
Note that  we still need the double quote in whatever is echo'ed to escape the angle brackets, so we need to include an escaped quote on either end of the string passed into `Util.echoN`.